bench/config/templates/frappe_sudoers

```diff
@@ -15,6 +15,6 @@
 {{ user }} ALL = (root) NOPASSWD: {{ nginx }}
 {% endif %}
 
-{{ user }} ALL = (root) NOPASSWD: /opt/certbot-auto
+{{ user }} ALL = (root) NOPASSWD: /usr/bin/certbot
```

Comment:
If certbot exists in bin, it exists in PATH. If that is true, you should use `bench.utils.which` instead. If nothing is found, you can pass this path into the template after verification.